### PR TITLE
OCPBUGS-23427: Updated Example load balancer configuration for user-p…

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.15 to 4.11

Issue:
[OCPBUGS-23427](https://issues.redhat.com/browse/OCPBUGS-23427)

Link to docs preview:
* [Example load balancer configuration for user-provisioned clusters - Bare metal](https://70837--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-bare-metal#installation-load-balancing-user-infra-example_installing-bare-metal)
* [Example load balancer configuration for user-provisioned clusters - Agnostic](https://70837--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_platform_agnostic/installing-platform-agnostic#installation-load-balancing-user-infra_installing-platform-agnostic)
* [Example load balancer configuration for user-provisioned clusters - IBM Z](https://70837--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z#installation-load-balancing-user-infra_installing-ibm-z)
* [Example load balancer configuration for user-provisioned clusters - IBM Power](https://70837--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power#installation-load-balancing-user-infra_installing-ibm-power)
* [Example load balancer configuration for user-provisioned clusters - OpenStack](https://70837--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-custom#installation-load-balancing-user-infra-example_installing-openstack-installer-custom)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by Timdawson264** [comment]
LGTM

File: modules/installation-load-balancing-user-infra.adoc [hunk 1]
```diff
@@ -213,10 +213,13 @@ listen api-server-6443 <1>
 listen machine-config-server-22623 <3>
   bind *:22623
   mode tcp
-  server bootstrap bootstrap.ocp4.example.com:22623 check inter 1s backup <2>
-  server master0 master0.ocp4.example.com:22623 check inter 1s
-  server master1 master1.ocp4.example.com:22623 check inter 1s
-  server master2 master2.ocp4.example.com:22623 check inter 1s
+  option  httpchk GET /readyz HTTP/1.0 <2>
```
Comment (ousleyp) [inline modules/installation-load-balancing-user-infra.adoc:216]: Should the `<2>` callout on the `option  httpchk GET /readyz HTTP/1.0` line be removed? I don't know anything about this, but it doesn't say "bootstrap" like the other lines that use that callout. I apologize if this is my ignorance talking. :)

**Review by ousleyp — COMMENTED** [review]
Just one comment. Thanks, Darragh! Please re-add the merge-review-needed label when you're ready.

File: modules/installation-load-balancing-user-infra.adoc (hunk 1 above)
Comment (Timdawson264) [reply modules/installation-load-balancing-user-infra.adoc:216]: The <2> callout should only be on the bootstrap lines.

Comment (dfitzmau) [reply modules/installation-load-balancing-user-infra.adoc]: Thanks, @ousleyp . Well spotted. I removed the callout reference for that line. Thanks Tim for chiming in!

**Pull request merged**